test/protobuf/json/rfc3339_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule Protobuf.JSON.RFC3339Test do
```

Comment:
The tests here are pretty thin but I didn't want to spend a couple of days writing tests that essentially do the same thing as all of the existing conformance tests 😄 We can add tests as we go here but I like this as a start.